tests/test_dsp_deconv.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import pytest
+import numpy as np
+import numpy.testing as npt
+import pyfar as pf
+from pyfar.signals import impulse
+import pyfar.signals as pfs
+
+from pyfar.dsp.dsp import deconvolve
+
+
```

Comment:
I'd suggest to add a `test_default`, which checks the correct results (incl. fft_norm etc.) for calling `deconvolve` with default parameters.

Reply:
good idea, I will add a `test_default`.